packages/gutenpride-template/src/style.css.mustache

```diff
@@ -0,0 +1,10 @@
+@font-face {
+    font-family: Gilbert;
+    src: url(./assets/gilbert-color.otf);
```

Comment:
Assets are copied to `assets` folder that is at the same level as `src` in the generated plugin:
```suggestion
    src: url(../assets/gilbert-color.otf);
```

Reply:
@jessynd This also answers your question regarding the assets folder. The folder won't actually show up after you have build the block from the template. It is only for the template itself :)

Reply:
Done!

Reply:
I don't have enough context but I can explain what happens here. There are two folders in the scaffolded block:
- `assets` with the font
- `src` with JS and CSS files

`wp-scripts build` or `wp-scripts start` they start processing through `src/index.js`, so they need to be able to find a relative path to the font, that's why it goes one level up.

In the `build` folder, there is `fonts` folder create by webpack, and the font is probably copied there (I don't know webpack code).